src/utils/vc/crypto/Bls12381BBSSignatureDock2022.js

```diff
@@ -18,8 +18,9 @@ import CustomLinkedDataSignature from './custom-linkeddatasignature';
 
 const SUITE_CONTEXT_URL = 'https://www.w3.org/2018/credentials/v1';
 
-const DEFAULT_PARSING_OPTS = {
-  useDefaults: true,
+export const DEFAULT_PARSING_OPTS = {
+  // useDefaults: true,
```

Comment:
Why was this changed?

Reply:
I think @SamHellawell changed it when he was debugging the issue.

Reply:
was for debugging schemas at one point but I think it makes sense to have use defaults as false now because we have schema generation - so defaults are never used